libs/fabric/src/lib/components/calendar/directives/calendar-strings-directive.component.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Directive, Input } from '@angular/core';
+import { ICalendarStrings } from 'office-ui-fabric-react';
+
+/**
+ * Wrapper directive for calendar strings
+ */
+@Directive({ selector: 'fab-calendar > fab-calendar-strings' })
+export class CalendarStringsDirective {
```

Comment:
Looks good.
Can you please add an example usage in the demo app?

Reply:
Example added to demo.